first_lessons_dushanbe/lesson_array.py

```diff
@@ -1,12 +1,15 @@
 #!/usr/bin/env python3
 # -*- coding: utf-8 -*-
 
+from define_count_and_index import fn_index
+
+
 def find_number(a, b):
     if a is None or b is None:
         return -1
 
-    if a.count(b) > 0:
-        return a.index(b)
+    if fn_index(a, b):
```

Comment:
you does not need to call `fn_index` twice. you can you it once.

Reply:
x = fn_index(a, b)
    if x >= 0:
        return x
    else:
        return -1

is it OK? or I'm thinking wrong?

Reply:
yes it is correct